File: LeetCode/Arrays/C++/2011. Final Value of Variable After Performing Operations.cpp
#include <bits/stdc++.h>
using namespace std;

class Solution {
public:
    int finalValueAfterOperations(vector<string>& operations) {
        int ans{};
        for(int i{};i<operations.size();i++)
        {
            if(operations[i][0]=='-' || operations[i][2]=='-') ans--;
            else ans++;
        }
        return ans;
    }
};

File: LeetCode/Arrays/JS/2011. Final Value of Variable After Performing Operations.js
/**
 * @param {string[]} operations
 * @return {number}
 */
var finalValueAfterOperations = function(operations) {
    let count=0;
    for(let i=0;i<operations.length;i++)
    {
        if(operations[i][0]=='-' || operations[i][2]=='-') count--;
        else count++;
    }
    return count;
};